fundingmanager.go

```diff
@@ -339,7 +350,7 @@ func (f *fundingManager) handleFundingRequest(fmsg *fundingRequestMsg) {
 			Code:             lnwire.ErrorMaxPendingChannels,
 			PendingChannelID: fmsg.msg.ChannelID,
 		}
-		fmsg.peer.queueMsg(errMsg, nil)
+		f.sendMessages(fmsg.peerAddress.IdentityKey, errMsg)
```

Comment:
The `sendMessages` function should also return an error so that the `fundingManager` can tell when a send failed and act accordingly.